zk/metadata.csv

```diff
@@ -1,30 +1,29 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+zookeeper.approximate_data_size,gauge,,,,,0,zookeeper,approx data size
+zookeeper.avg_latency,gauge,,millisecond,,The amount of time it takes for the server to respond to a client request.,0,zookeeper,avg latency
 zookeeper.bytes_received,gauge,,,,Number of bytes received,0,zookeeper,
 zookeeper.bytes_sent,gauge,,,,Number of bytes sent,0,zookeeper,
-zookeeper.bytes_outstanding,gauge,,,,Number of bytes outstanding,0,zookeeper,
```

Comment:
This is the only line actually removed, the rest of the changes are because of I've sorted the metric names